include/boost/windows_sspi/detail/sspi_impl.hpp

```diff
@@ -107,17 +114,19 @@ class sspi_handshake {
     OutBuffer.pBuffers = OutBuffers;
     OutBuffer.ulVersion = SECBUFFER_VERSION;
 
+    DWORD out_flags = 0;
+
     m_last_error = detail::sspi_functions::InitializeSecurityContext(m_credentials,
                                                                      m_context,
                                                                      nullptr,
-                                                                     m_flags_in,
+                                                                     context_flags,
```

Comment:
Not an actionable item in the context of this PR: Might want to consider allowing a user to specify additional options in the future. IIRC, this is where you enable mTLS authentication.

Reply:
I think I only need to change some options in the call to `AcquireCredentialsHandle()` currently residing in the context class to support client certificates, but there is a flag called `ISC_REQ_USE_SUPPLIED_CREDS` which is documented to indicate that:

> Schannel must not attempt to supply credentials for the client automatically.

So it could indeed be relevant and I'll keep that in mind. The documentation is a bit vague to say the least, but I've tried to ensure that it should be fairly easy to write some tests for that once I start looking into certificate handling.

Thanks for pointing this out.